### PR TITLE
[deploy] enable install NTP service in deployment 

### PR DESCRIPTION
**Pull request by YanjieGao**
Why: In some environments (Such as Azure Ubuntu VM), the NTP server is not installed by default. This feature is provided to check and install the ntp service.

Which component needed it: Distributed system such as etcd inner PAI. 

[PAI Prerequisites](https://github.com/Microsoft/pai#1-prerequisites-)

### CODE REVIEW

File: deployment/k8sPaiLibrary/maintainlib/add.py [hunk 1]
```diff
@@ -93,6 +93,11 @@ def job_executer_add_work_node(self):
             self.logger.error("Failed to update the /etc/hosts on {0}".format(self.node_config['hostip']))
             sys.exit(1)
 
+        commandline = "sudo ./{0}/prerequisites-install.sh {0}".format(self.jobname)
+        if common.ssh_shell_with_password_input_paramiko(self.node_config, commandline) == False:
+            self.logger.error("Failed to install docker-ce on {0}".format(self.node_config['hostip']))
```
Comment (ydye) [inline deployment/k8sPaiLibrary/maintainlib/add.py]: docker ce?

File: deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template [hunk 2]
```diff
@@ -61,3 +61,5 @@ if [ -f "kubernetes.tar" ]; then
     rm -rf kubernetes.tar
 
 fi
+
+apt-get -y purge ntp
```
Comment (ydye) [inline deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template]: What if the ntp is installed by others? Why to unstall it?

File: deployment/k8sPaiLibrary/maintainlib/add.py (hunk 1 above)
Comment (YanjieGao) [reply deployment/k8sPaiLibrary/maintainlib/add.py]: yes, consider this, this is a little like install docker-ce and uninstall docker-ce type problem. If current PAI default not uninstall docker when clean k8s, i will remove this uninstall cmd and let user customized uninstall ntp? What's your suggestion?

File: deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template (hunk 2 above)
Comment (YanjieGao) [reply deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template]: yes, consider this, this is a little like install docker-ce and uninstall docker-ce type problem. If current PAI default not uninstall docker when clean k8s, i will remove this uninstall cmd and let user customized uninstall ntp? What's your suggestion?

File: deployment/k8sPaiLibrary/maintainlib/add.py (hunk 1 above)
Comment (ydye) [reply deployment/k8sPaiLibrary/maintainlib/add.py]: Please change the log.

File: deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template (hunk 2 above)
Comment (ydye) [reply deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template]: Don't uninstall it.

Comment (YanjieGao) [reply deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template]: removed

File: deployment/k8sPaiLibrary/maintaintool/ntp-install.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        run("echo server 0.rhel.pool.ntp.org >> /etc/ntp.conf")
+        run("echo server 1.rhel.pool.ntp.org >> /etc/ntp.conf")
+        run("echo server 2.rhel.pool.ntp.org >> /etc/ntp.conf")
+        run("echo server 3.rhel.pool.ntp.org >> /etc/ntp.conf")
+        print("NTP_SERVERS_INSTALLED")
+
+def restart_ntp_service(ntp_service):
+        print(just_run("service "+ntp_service+" restart"))
+
+def main():
```
Comment (ydye) [inline deployment/k8sPaiLibrary/maintaintool/ntp-install.py]: Why don't you write a shell script?

Comment (YanjieGao) [reply deployment/k8sPaiLibrary/maintaintool/ntp-install.py]: python is easy to extend to other os in one file in the future and better to catch each steps state

Comment (ydye) [reply deployment/k8sPaiLibrary/maintaintool/ntp-install.py]: Actually, all your command are shell. I don't think so.

**Pull request closed without merging**